src/redux/reducer.type.ts

```diff
@@ -5,10 +5,16 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
 
+import { ComputingType } from 'components/computing-status/computing-type';
+import { RunningStatus } from 'components/utils/running-status';
 import { UUID } from 'crypto';
```

Comment:
this import "import { UUID } from 'crypto'; "
in my IDE i have this message
"Module '"crypto"' has no exported member 'UUID' " 
the same for you ?